libpgcommon/lwgeom_cache.h

```diff
@@ -35,6 +35,18 @@
 /* Returns the MemoryContext used to store the caches */
 MemoryContext PostgisCacheContext(FunctionCallInfo fcinfo);
 
+typedef struct {
+	uint32_t count;
+	GSERIALIZED *geom;
+} SHARED_GSERIALIZED;
```

Comment:
Reverse struct entries to avoid padding between `count` and `geom`? (You're the expert on such optimizations though 😄

Reply:
Not really, it should be the other way around although in x86-64 it doesn't matter (it's 16 bytes no matter the order) and since it's not placed in any array or something like that it would really be a micro-tiny-optimization, but a good thing to keep in mind on principle. I'll change it.